README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
@stevvooe 
Thanks, I have updated it based on the suggested change, could you please review it and merge it?